js/ui/test-button.j.jsx

```diff
@@ -0,0 +1,72 @@
+import registerComponent from '../core/component_registrator';
+import Widget from './widget/ui.widget';
+import { extend } from '../core/utils/extend';
+import ButtonView from './test-button.p';
+import * as Preact from 'preact';
+
+class Button extends Widget {
+    _initMarkup() { }
+
+    _render() {
+        this._renderContent();
+    }
+
+    _renderContent() {
+        const options = this.option();
+        const container = this.$element().children().length === 0 /** isFirstRender*/ ? this.$element().get(0) : undefined;
```

Comment:
const isFirstRender = this.$element().children().length === 0 ?

Reply:
done